specs/BackgroundTask/BackgroundTaskBuilder.md

```diff
@@ -0,0 +1,98 @@
+Current [BackgroundTaskBuilder](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.background.backgroundtaskbuilder) is designed for UWP applications and many of the Background Task triggers are not supported for full trust COM Components and are supported only to be registered with Windows Runtime components that is to be launched with backgroundtaskhost process. Due to this, WinAppSDK desktop applications won’t be able to directly register the full trust COM components to be launched with background task triggers and have to do workaround of including the Windows Runtime components in the project.
+This API takes care of avoiding this workaround so that WinAppSDK applications can directly register the full trust COM components with the background tasks.
+
+# Conceptual pages (How To)
+
+ ## Register Background Task (c++)
+
+ Registering a Background Task for full trust COM component. Developers may also need to set the UniversalBGTask property as true in their project configuration.
+
+ ```c++
+       //Using WinAppSDK API for BackgroundTaskBuilder
+        winrt::Microsoft::Windows::ApplicationModel::Background::BackgroundTaskBuilder builder;
+        SystemTrigger trigger = SystemTrigger(SystemTriggerType::TimeZoneChange, false);
+        auto backgroundTrigger = trigger.as<IBackgroundTrigger>();
+        builder.SetTrigger(backgroundTrigger);
+        builder.SetTaskEntryPointClsid(classGuid);
+        builder.Register(); 
+```
+
+# API Pages
+
+## BackgroundTaskBuilder class
+
+Represents a background task to register with the system.
+
+This class is not agile, which means that you need to consider its threading model and marshaling behavior. For more info, see [Threading and Marshaling (C++/CX)]( https://learn.microsoft.com/en-us/cpp/cppcx/threading-and-marshaling-c-cx?view=msvc-170 ) and [Using Windows Runtime objects in a multithreaded environment (.NET)]( https://learn.microsoft.com/en-us/windows/uwp/threading-async/using-windows-runtime-objects-in-a-multithreaded-environment)
+
+
+## BackgroundTaskBuilder constructor
+
+| Name | Description |
+|-|-|
+| BackgroundTaskBuilder  | Create a new BackgroundTaskBuilder object.|
+
+
+## BackgroundTaskBuilder Properties
+
+| Name | Description | Value |
+|-|-|-|
+|Name| Gets or sets the name of a background task.|Name(A string representing name of the task) |
+|TaskGroup| Gets and sets the group identifier.| TaskGroup (The group identifier.) |
+
+
+## BackgroundTaskBuilder Methods
+
+| Name | Description | Parameters | Returns |
+|-|-|-|-|
+| AddCondition(IBackgroundCondition) | Adds a condition to a background task.| __IBackgroundCondition__ Condition for Background Task | |
```

Comment:
Please include an example of AddCondition method.

Reply:
Added example